src/include/daos/tests_lib.h

```diff
@@ -173,6 +173,15 @@ typedef struct {
 	char		host[50];
 }  device_list;
 
+#define MAX_TEST_TARGETS_PER_POOL 10
+#define MAX_TEST_RANKS_PER_POOL 5
+typedef struct {
+	uuid_t		pool_id;
+	int		rank[MAX_TEST_RANKS_PER_POOL];
+	int		tgts[MAX_TEST_TARGETS_PER_POOL];
+	char		host[50];
+}  pool_table_list;
+
```

Comment:
According to our discussion, we can easily know the total node count and total target count from pool query, then we can calculate the per node target count by (total target count / total node count). This data structure won't be necessary anymore?